numpy/core/src/multiarray/einsum.dispatch.c.src

```diff
@@ -0,0 +1,2071 @@
+/*
+ * This file contains the implementation of the 'einsum' function,
+ * which provides an einstein-summation operation.
+ *
+ * Copyright (c) 2011 by Mark Wiebe (mwwiebe@gmail.com)
+ * The University of British Columbia
+ *
+ * See LICENSE.txt for the license.
+ */
+/**
+ * @targets $maxopt baseline
```

Comment:
Everything is starting from here, we call it the [configuration statements](https://numpy.org/devdocs/reference/simd/simd-optimizations.html#dispatch-able-sources-and-configuration-statements).
The following explanations not combined on the doc yet:

'$maxopt': is a policy that forces the compiler to add maximum acceptable of the optimization level(-O3 in gcc).
 
`(AVX2 FMA3)`: is a multi-target that gathers two features since in gcc/clang AVX2 and FMA3 don't imply one another, [see features tables](https://numpy.org/devdocs/html/reference/simd/simd-optimizations.html#x86-cpu-feature-names).

Reply:
another thing to add, by default `CCompilerOpt` will skip any requested targets if its part of the baseline.
in other words, it will skip `sse2` according to the default configuration.